src/resultsdisassemblypage.h

```diff
@@ -91,4 +94,16 @@ public slots:
     QString m_objdump;
     // Map of symbols and its locations with costs
     Data::DisassemblyResult m_disasmResult;
+    // Not to show machine codes of Disassembly
+    bool m_noShowRawInsn;
+    // Not to show address of Disassembly
+    bool m_noShowAddress;
+    // Switch Assembly to Intel Syntax
+    bool m_intelSyntaxDisassembly;
+    // Setter for m_noShowRawInsn
```

Comment:
remove useless comments, move functions above the member variables to the other functions